var/spack/repos/builtin/packages/py-numba/package.py

```diff
@@ -12,18 +12,22 @@ class PyNumba(PythonPackage):
     homepage = "https://numba.pydata.org/"
     url      = "https://pypi.io/packages/source/n/numba/numba-0.35.0.tar.gz"
 
+    version('0.48.0', sha256='9d21bc77e67006b5723052840c88cc59248e079a907cc68f1a1a264e1eaba017')
     version('0.40.1', sha256='52d046c13bcf0de79dbfb936874b7228f141b9b8e3447cc35855e9ad3e12aa33')
     version('0.35.0', sha256='11564937757605bee590c5758c73cfe9fd6d569726b56d970316a6228971ecc3')
 
+    depends_on('python@3.6:', type=('build', 'run'), when='@0.48.0:')
     depends_on('python@3.3:3.7.9999', type=('build', 'run'), when='@0.40.1:')
     depends_on('python@3.3:3.6.9999', type=('build', 'run'), when='@:0.35.0')
+    depends_on('py-numpy@1.15:', type=('build', 'run'), when='@0.48.0:')
     depends_on('py-numpy@1.10:1.99', type=('build', 'run'))
     depends_on('py-setuptools', type='build')
```

Comment:
```suggestion
    depends_on('py-setuptools', type=('build', 'run'))
```
`numba/entrypoints.py` has an import of `pkg_resources`, which is provided by `py-setuptools`.